src/toolchain-conventions.adoc

```diff
@@ -216,6 +216,41 @@ As of November 2021 the additional tail-call entry points are only
 implemented in compiler-rt, and calls will only be generated by LLVM
 when the option `-mllvm -save-restore-tailcall` is specified.
 
+== Profile-based format
+
+Profiles should be recognized and used in the `-march=` option. The benefit of using
+the `-march` option is easy for toolchain parsing the profiles string and expanding
+it into normal extensions combinations. 
+
+Profiles format has the following BNF form `"-march="<profile-name>"_"[option-ext]*`.
+
+`profile-name ::= "RV"<profile-family-name><profile-ratified-year><privilege-mode><ISA-XLEN>`
+
+`profile-family-name ::= "i" | "m" | "a"`
+
+`profile-ratified-year-version ::= "20" | "22" | "23A" | "23B"`
+
+`privilege-mode ::= "u" | "s" | "m"`
+
+`ISA-XLEN ::= "64" | "32"`
+
+`option-ext ::= 'a legal RISC-V extension name'`
+
+As the spec defines, to use the profiles it should follow profile naming convention
+(See [3.4 form spec doc](https://github.com/riscv/riscv-profiles)), the toolchain
+will check whether an input profile name is correct at first, then do the parse
+work.
+
+To distinguish between ordinary extension input and input with profiles,
+profiles are assumed to be entered *at the beginning of the -march option*, and
+then input other extensions. Profiles should input in the `-march` option.
+
+Examples:
+
+`-march=rvi20u64 -mabi=lp64` equals `-march=rv64i -mabi=lp64`
+
+`-march=rva22u32` equals `-march=rv64gcb_zic64b_zicbom_zicbop_zicboz_ziccamoa_ziccif_zicclsm_ziccrse_zicntr_zihpm_za64rs_zfhmin_zkt`
```

Comment:
Add one more example to demo the syntax of `-march=<profile-name>_<option-ext>`

Reply:
Adds a new example with this form.